### PR TITLE
Filter fasta sequences to prevent downstream errors

### LINKED ISSUE #169: Filter fasta entries that contain invalid amino acids (X)

### PR DESCRIPTION
**Pull request by susannasiebert**
This fixes #169 and #168

### CODE REVIEW

File: pvacseq/lib/pipeline.py [hunk 1]
```diff
@@ -297,6 +303,17 @@ def __init__(self, **kwargs):
         Pipeline.__init__(self, **kwargs)
         self.peptide_sequence_length = 31
 
+    def generate_fasta(self):
+        print("Generating Variant Peptide FASTA File")
+        sys.stdout.flush()
+        lib.generate_fasta.main([
+            self.tsv_file_path(),
+            str(self.peptide_sequence_length),
+            '9',
```
Comment (tmooney) [inline pvacseq/lib/pipeline.py]: I found the rationale for the `9` in #168, but it might be nice to document its significance in some way.

Comment (susannasiebert) [inline pvacseq/lib/pipeline.py]: I added comment to this line of code.

**Comment by tmooney** [comment]
+1 🎊

**Comment by acoffman** [comment]
+1

**Pull request merged**